scripts/setup-google-adc.sh

```diff
@@ -3,10 +3,10 @@
 #  1. `gcloud auth login <gcp-email>` and authenticate
 #  2. `gcloud auth application-default login` and authenticate
 #  3. `cat ~/.config/gcloud/application_default_credentials.json` and copy the output
-#  4. Go to https://gitpod.io/environment-variables/ and create:
```

Comment:
**thought:** although this already redirects to `/settings`, it's best to use a URL that will probably be around in the long run.